### PR TITLE
don't check password in native mode if --output-database is not true

### LINKED ISSUE #563: synopsysctl native should only require DB credentials with --output-database
**How to reproduce it (as minimally and precisely as possible)**:
./synopsysctl create blackduck native  test
ERRO[0000] synopsyctl failed: required flag(s) "admin-password", "postgres-password", "user-password" not set

### PR DESCRIPTION
**Pull request by yashbhutwala**
/closes #563

### CODE REVIEW

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/573#" title="Author self-approved">yashbhutwala</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/master/OWNERS)~~ [yashbhutwala]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: pkg/synopsysctl/cmd_create.go [hunk 1]
```diff
@@ -190,13 +190,50 @@ var createAlertNativeCmd = &cobra.Command{
 	},
 }
 
+func checkPasswords(cmd *cobra.Command) {
```
Comment (mphammer) [inline pkg/synopsysctl/cmd_create.go]: Move this function under the comment for `Create Black Duck Commands`

File: pkg/synopsysctl/cmd_create.go [hunk 2]
```diff
@@ -190,13 +190,50 @@ var createAlertNativeCmd = &cobra.Command{
...
 /*
 Create Black Duck Commands
 */
 
 var createBlackDuckPreRun = func(cmd *cobra.Command, args []string) error {
 	// Check the user's flags
 	err := createBlackDuckCtl.CheckSpecFlags(cmd.Flags())
+
+	// if the caller is createBlackDuckNativeCmd
+	if cmd.Use == "native NAME" {
```
Comment (mphammer) [inline pkg/synopsysctl/cmd_create.go]: You could do each of these checks in the ArgsE of each command instead of having this check here.

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/573#pullrequestreview-254816177" title="Approved">mphammer</a>*, *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/573#" title="Author self-approved">yashbhutwala</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/master/OWNERS)~~ [mphammer,yashbhutwala]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: pkg/synopsysctl/cmd_create.go (hunk 1 above)
Comment (yashbhutwala) [reply pkg/synopsysctl/cmd_create.go]: done.

File: pkg/synopsysctl/cmd_create.go (hunk 2 above)
Comment (yashbhutwala) [reply pkg/synopsysctl/cmd_create.go]: good suggestion, done.

File: pkg/synopsysctl/cmd_create.go [hunk 3]
```diff
@@ -250,6 +276,7 @@ var createBlackDuckCmd = &cobra.Command{
 			cmd.Help()
 			return fmt.Errorf("this command takes 1 argument")
 		}
+		checkPasswords(cmd)
```
Comment (msenmurugan) [inline pkg/synopsysctl/cmd_create.go]: Instead of cmd, can you pls pass only the flags

Comment (yashbhutwala) [reply pkg/synopsysctl/cmd_create.go]: done

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/573#pullrequestreview-254816177" title="Approved">mphammer</a>*, *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/573#pullrequestreview-254831872" title="LGTM">rrati</a>*, *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/573#" title="Author self-approved">yashbhutwala</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/master/OWNERS)~~ [mphammer,rrati,yashbhutwala]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/573#pullrequestreview-254816177" title="Approved">mphammer</a>*, *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/573#pullrequestreview-254849592" title="Approved">msenmurugan</a>*, *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/573#pullrequestreview-254831872" title="LGTM">rrati</a>*, *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/573#" title="Author self-approved">yashbhutwala</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/master/OWNERS)~~ [mphammer,msenmurugan,rrati,yashbhutwala]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/573#pullrequestreview-254816177" title="Approved">mphammer</a>*, *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/573#issuecomment-506028930" title="LGTM">msenmurugan</a>*, *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/573#pullrequestreview-254831872" title="LGTM">rrati</a>*, *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/573#" title="Author self-approved">yashbhutwala</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/master/OWNERS)~~ [mphammer,msenmurugan,rrati,yashbhutwala]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by operator-r2d2** [comment]
@yashbhutwala: once the present PR merges, I will cherry-pick it on top of release-2019.6.x in a new PR and assign it to you.

<details>

In response to [this](https://github.com/blackducksoftware/synopsys-operator/pull/573#issuecomment-506030782):

>/cherrypick release-2019.6.x


Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Comment by operator-r2d2** [comment]
@yashbhutwala: new pull request created: #578

<details>

In response to [this](https://github.com/blackducksoftware/synopsys-operator/pull/573#issuecomment-506030782):

>/cherrypick release-2019.6.x


Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Pull request merged**